design/deployment_design.md

```diff
@@ -0,0 +1,104 @@
+#  **Dockerless CLI -** deployment
+
+This document is the design document for the deployment of the Python CLI. 
+
+Currently, our Python CLI can be deployed in the following ways:
+
+1. Install the CLI via the pip Python package manager like so: `pip3 install conjur-client`. **Python is required** on machine if installing the CLI in this way. If installed via pip, the user can then run the CLI like so `conjur-cli --help`
+
+2. Install and run the executable (**Python is** **not** required on the machine)
+
+   The CLI is delivered as a standalone executable, packed with the required runtime and dependencies (excluding a few base linked libraries that are expected on the system). Therefore, there is no need for Python or other prerequisites to be installed on machine. The executable can be fetched by navigating to our [Python CLI release page](https://github.com/cyberark/conjur-api-python3/releases/tag/v0.0.5). If installed in this manner the CLI can be invoked by, `./python-cli-executable-location variable get ...`
+
+### Design
+
+#### Pip3 package
+
+We make CLI package available to the public by pushing to a public repository called PyPi. I propose that we maintain pushing our the Python CLI to pip3 for both customer's machines have have internet access and for our development work. 
+
+Our Python API is also delivered this way and PyPi is the main installation method for it so this should be retained. The infrastructure is already in-place so this would be at no additional effort.
```

Comment:
Python API is our SDK?

Reply:
@Tovli Yes, it's a wording issue. Our repos are called "conjur-api-<language>" but effectively it's an SDK for that language. In this case, when you see "Python API", what's really meant is "Python SDK" or "Python Client Library"